src/Workspaces/Core/Portable/Workspace/Workspace.cs

```diff
@@ -171,8 +172,8 @@ protected Solution SetCurrentSolution(Solution solution)
 
     /// <summary>
     /// Sets the <see cref="CurrentSolution"/> of this workspace. This method does not raise a <see
-    /// cref="WorkspaceChanged"/> event.  This method should be used <em>sparingly</em>.  As much as possible,
-    /// derived types should use the SetCurrentSolution overloads that take a transformation.
+    /// cref="WorkspaceEventType.WorkspaceChange"/> event.  This method should be used <em>sparingly</em>.
```

Comment:
We may need to suppress the pragma here -- I'm not sure if this will trip up any documentation generators since this is now going to an internal type instead.

Reply:
bah, the cref isn't that important. The comment is pretty clear without it.